byu-calendar-row/byu-calendar-row.scss

```diff
@@ -152,17 +153,20 @@ $ticket-region-width: 250px;
 	}
 
 	#month-name {
-		padding: 10px 0px;
+		margin: 10px 0px;
 		font-size: 13px;
 		text-transform: uppercase;
-		margin-bottom: 0px;
 		font-weight: 500;
+		height: 13px;
```

Comment:
To fix the padding problem the height specification needed to be deleted. Will be included in a patch in a second. Probably it would be easier to just push up a new commit now that I think about it.

Reply:
correct - push up another commit is the correct way to adjust the code.